crates/telio-wg/src/wg.rs

```diff
@@ -748,7 +867,53 @@ impl State {
 
         self.interface = to.clone();
 
-        success
+        Ok(success)
+    }
+
+    fn compute_new_node_info(
+        &self,
+        old_info: &NoLinkDetectionInfo,
+        new_tx: u64,
+        new_rx: u64,
+    ) -> NoLinkDetectionInfo {
+        let configurable_rtt = match self.no_link_detection {
+            NoLinkDetection::Disabled => DEFAULT_CONFIGURABLE_RTT,
+            NoLinkDetection::Enabled { rtt } => rtt,
+        };
```

Comment:
IMHO this doesn't make sense when the `NoLinkDetection` is disabled, maybe `rtt` should be passed as a parameter (then self is unused) and the `match thing should be higher in the call stack